packages/@atjson/offset-annotations/src/utils/social-urls.ts

```diff
@@ -349,13 +349,34 @@ function normalizeTelegramUrl(url: IUrl) {
     },
   };
 }
+function isRedditURL(url: IUrl) {
+  return (
+    (url.host === "www.redditmedia.com" && url.pathname.startsWith("/r/")) ||
+    url.host === "www.reddit.com"
+  );
+}
 
-export function identify(
-  url: IUrl
-): {
+function normalizeRedditURL(url: IUrl) {
+  let ref_source = "embed";
+  let ref = "share";
+  let embed = true;
+  return {
+    attributes: {
+      url: `https://www.redditmedia.com${url.pathname}?ref_source=${ref_source}&amp;ref=${ref}&amp;embed=${embed}`,
+      width: "640",
+      height: "141",
```

Comment:
Is there a reason we are setting a static size instead of reading it from the snippet?

Reply:
Oh nvm I think this is a default size (since we are identifying purely by the URL here, not an embed shippet), but in the html conversion we will override this based on the `height` and `width` attributes on the iframe

Reply:
Please add a test for that in the html converter to make sure this is working properly

Reply:
No this was only to check working workflow. will change it.